### ISSUE TITLE
PHP Warning bei Screenshot Anhang bei Turnierergebnissen 

### ISSUE THREAD

**Issue by daencore** [lansuite/lansuite#348]
Wenn man in einem Turnier eine Spielergebnis einträgt und dabei einen Screenshot (jpeg) anhängt kommt folgende Fehlermeldung: `PHP Warning:unlink(ext/tournament_screenshot/1png): No such file or directory in /www/htdocs/w00c2db7/lansuite/modules/tournament2/submit_result.php on line 253` 

![whatsapp image 2018-09-07 at 08 47 17](https://user-images.githubusercontent.com/39373552/45212077-4659e300-b294-11e8-93fb-8b121478c1b0.jpg)

Das Angehängte Bild wird Trotzdem Hochgeladen und das Ergebnis eingetragen. Jemand eine Idee zu dem Fehler? Danke für Hilfe.

**Comment by M4LuZ** [comment]
Yup. Es gibt anscheinend genau ein Foto pro Match und LS versucht das beim Upload zu löschen.
Auch wenn das der erste Upload ist und dementsprechend noch kein Foto für das Match existieren kann...

**Comment by M4LuZ** [comment]
Should be fixed with PR #351.
Would be great if you could give it a try, I would close this in a week or so otherwise.

**Comment by daencore** [comment]
Hey, Ich habe auf die aktuelle Master geupdatet. Die Vorhandenen Ergebnisse werden jetzt richtig gereiht. Danke

Jedoch habe ich jetzt ein anderes Schwerwiegenderes Problem durch das Update bekommen.

Wenn ich ein Turnier generiert habe und dann das erste Spielergebnis eintrage bekomme ich folgende Fehlermeldung: 
![bild 1 rfehler](https://user-images.githubusercontent.com/39373552/45445779-73aaf480-b6cb-11e8-8c6c-c15be12eb817.jpg)

Danach beim Zweiten Spielergebniss diesen fehler: 
![bild 2bfsbsf](https://user-images.githubusercontent.com/39373552/45445823-91785980-b6cb-11e8-835a-3234100ede74.jpg)

Im Endeffekt ist der Turnierbaum dann Brocken. Was könnte das sein?
![bild 3dbfbfd](https://user-images.githubusercontent.com/39373552/45445850-a8b74700-b6cb-11e8-9939-4e65ed0255d2.jpg)

Bitte um Hilfe. Danke

**Comment by M4LuZ** [comment]
Erm...
Die Änderung bezüglich Single-Ele ranking ist noch in in PR  #350 und aktuell noch gar nicht im Master.
Also was auch immer das Problem ist, damit hatte es nichts zu tun ;)

Und bitte die Fehler in separaten Tickets auflisten oder generalisieren, sonst blicken wir leider nicht mehr durch, was wo behandelt wurde.
Ich entnehme dem aber, dass das Problem des Pic-uploads hiermit durch ist?

**Comment by daencore** [comment]
Danke für die Antwort. Ok ich dachte es ist im Master. Naja habe deinen Code manuel eingefügt und der Bild Upload Funktioniert soweit ohne Probleme. 

Für den neuen Fehler erstelle ein neues Ticket.
